### ISSUE TITLE
UX: Don't show fee slider on rinkeby

### ISSUE THREAD

**Issue by ligi** [balance-io/balance-manager#380]
on rinkeby the fee slider makes no sense - should always be 21Gwei here

![selection_106](https://user-images.githubusercontent.com/111600/43419220-a00c8a8c-9440-11e8-905e-5a0143056674.png)

**Comment by ricburton** [comment]
Good to know—thanks! Right now we are not focused on developers but this seems like a quick fix.

Richard

**Comment by ligi** [comment]
I do not think this is only important for developers - users might also want to try out things before using it with real money. Especially when it is in a early development state like this.
